### PR TITLE
Rewrite SHOW SERIES to SELECT

### LINKED ISSUE #5580: [0.10.0] SHOW SERIES only shows series from certain nodes
When using a 3-node cluster with a replication factor of 1, some nodes do not appear to be consulting all nodes for series information.

Node 1:

```
> create database db
> use db
Using database db
> alter retention policy default on db REPLICATION 1
> insert cpu value=1
> show measurements
name: measurements
------------------
name
cpu

> show series
>
```

Node 2 (after all commands on node 1 completed):

```
> use db
Using database db
> show measurements
name: measurements
------------------
name
cpu

> show series
name: cpu
---------
_key
cpu

>
```

Node 3 (after all commands on node 1 completed):

```
> use db
Using database db
> show measurements
name: measurements
------------------
name
cpu

> show series
>
```

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

This pull request rewrites `SHOW SERIES` statements into `SELECT` statements so they can be execute by the query engine.

Fixes #5580

- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: tsdb/shard.go [hunk 1]
```diff
@@ -882,6 +884,84 @@ func (itr *MeasurementIterator) Next() *influxql.FloatPoint {
...
+		fields: opt.Aux,
+	}, nil
+}
+
+// Close closes the iterator.
+func (itr *seriesIterator) Close() error { return nil }
+
+// Next emits the next point in the iterator.
+func (itr *seriesIterator) Next() *influxql.FloatPoint {
+	for {
```
Comment (e-dard) [inline tsdb/shard.go]: This loop will only iterate once.

**Comment by e-dard** [comment]
LGTM. Have you tested it on a local cluster @benbjohnson ?

File: tsdb/shard.go (hunk 1 above)
Comment (benbjohnson) [inline tsdb/shard.go]: Good point. :)

Comment (benbjohnson) [inline tsdb/shard.go]: Fixed in ebf7cf1.

**Comment by benbjohnson** [comment]
@e-dard Yes, I tested it in a local cluster and it works.

**Pull request merged**